tests/baselines/reference/user/async.log

```diff
@@ -94,7 +94,6 @@ node_modules/async/dist/async.js(622,80): error TS2339: Property 'nodeType' does
 node_modules/async/dist/async.js(748,84): error TS2339: Property 'nodeType' does not exist on type 'NodeModule'.
 node_modules/async/dist/async.js(754,49): error TS2339: Property 'process' does not exist on type 'false | (Global & typeof globalThis)'.
   Property 'process' does not exist on type 'false'.
-node_modules/async/dist/async.js(810,45): error TS2554: Expected 0 arguments, but got 1.
```

Comment:
`var isBuffer = nativeIsBuffer || stubFalse` used to take `any | () => false` and reduce to `() => false`. Not it reduces to `any`. Seems correct-er-ish.